README.rst

```diff
@@ -0,0 +1,62 @@
+Exasol Toolbox
+==============
+
+Your one-stop solution for managing all standard tasks and core workflows of your Python project.
+
+.. image:: https://github.com/exasol/python-toolbox/actions/workflows/ci.yml/badge.svg?branch=main
+   :target: https://github.com/exasol/python-toolbox/actions/workflows/ci.yml
+   :alt: Checks Main
+
+.. image:: https://img.shields.io/pypi/l/exasol-toolbox
+   :target: https://opensource.org/licenses/MIT
+   :alt: License
+
+.. image:: https://img.shields.io/pypi/dm/exasol-toolbox
+   :target: https://pypi.org/project/exasol-toolbox/
+   :alt: Downloads
+
+.. image:: https://img.shields.io/pypi/pyversions/exasol-toolbox
+   :target: https://pypi.org/project/exasol-toolbox/
+   :alt: Supported Python Versions
+
+.. image:: https://img.shields.io/pypi/v/exasol-toolbox
+   :target: https://pypi.org/project/exasol-toolbox/
+   :alt: PyPi Package
+
+🚀 Features
+-----------
+
+- Centrally managed standard tasks:
+  - code formatting & upgrading
+  - linting
+  - type-checking
+  - unit-tests
+  - integration-tests
+  - coverage
+  - documentation
+
+- Centrally managed core workflows:
+  - workspace/project verification
+  - build and publish releases
+  - build and publish documentation
+
+- Configurable & Extensible:
+  - Project configuration
+  - Event hooks
+
+🔌️ Prerequisites
+-----------------
+
+- `Python <https://www.python.org/`__ >= 3.9
```

Comment:
purposefully added bug here for test. see https://github.com/exasol/python-toolbox/pull/557/files#r2313288794.

after setting up that test, please fix this so that the test_works_as_expected works as expected.